### ISSUE TITLE
Pad midi note C-0

### ISSUE THREAD

**Issue by atmtt** [polyend/TrackerIssues#679]
Hello! I've try to play/record external gear with the Tracker's pads. I've changed the midi root note but is impossible to play under C-1. Is it possible extend the range of the pads to the lowest note C-0? Thanks!

**Comment by danielruizmunoz** [comment]
Yes, this is a TOP improvement and I think it's very easy to implement. For example, sequencing the Digitakt, with runs the "no notes" sounds starting in C-0, can't be done properly right now.
